Sources/Tools/plShaderAssembler/main.cpp

```diff
@@ -133,21 +155,12 @@ static void IAssShader(const plDXShaderAssembler& ass, const char* name)
     plFileName outFile = ST::format("{}.h", varName);
 
     ST::printf("Processing {} into {}\n", name, outFile);
+
     hsUNIXStream outFp;
-    if (!outFp.Open(outFile, "w")) {
-        fputs("%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n", stderr);
-        ST::printf(stderr, "Error opening file {} for output\n", outFile);
-        fputs("%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n", stderr);
-        return;
-    }
+    if (AttemptToOpenOutFile(outFile, outFp)) return;
```

Comment:
Plasma Code Style prohibits this formatting.

```suggestion
    if (AttemptToOpenOutFile(outFile, outFp))
        return;
```